### ISSUE TITLE
Video metadata is not removed

### ISSUE THREAD

**Issue by snooppr** [deckerst/aves#927]
Not sure "feature request" or "bug".

The bottom line: Aves can remove metadata from photos, and the “remove video metadata” option is also available, but the item is inactive.  I once chose Aves to replace another application, in which the functionality reported that it could and deleted metadata, but in fact it was a vulnerability that has not been fixed until now.

------

![video metadata](https://github.com/deckerst/aves/assets/61022210/933e63e9-6297-41b4-955d-04f53280b348)
A simple video, screencast of a desktop on Android.  Mat2 says that the video has tags: manufacturer and model of the gadget (shaded), and version Android.

-----

![aves no clear metadata video](https://github.com/deckerst/aves/assets/61022210/8e4ed0fb-4cbc-46bf-9de2-68be44db5682)
Aves also sees this metadata, but it cannot clear it, the button is inactive.

-----

![mat2 clear metadata video](https://github.com/deckerst/aves/assets/61022210/463c6d3c-14d9-4e78-add8-c31708ab8834)
Mat2 has successfully cleared the tags/metadata from the video and reports that everything is clean.

----

Simply put, teach Aves to remove metadata and tags from videos, not just photos.  On Android, the Mat2 utility from the Termux environment does this well.

**Comment by Robert321245** [comment]
Making this available would be amazing since I am not aware of any other FOSS application capable to remove video metadata, only image.

Removing just metadata related to manufactor, time, and geolocation would be enough for privacy and would not change the video (color profile and other stuff).

**Comment by lei12** [comment]
+1
Aves messes up the files order in other Apps while imposing an inappropriate metadata date for adding a tag to a video on which metadata are initially missing. The metadata date ends up being today, when it should be the initial creation date of the file. And we have no option to avoid that, nor to get back to the original state. Adding a tag to a video should not affect its metadata date. It is not editing the images, it is simply adding information in the background.
Please let us remove the metadata date added by Aves, or let us choose which date we want it to apply.

**Comment by deckerst** [comment]
> The metadata date ends up being today, when it should be the initial creation date of the file

@lei12 that does not sound right. Can you file an issue for this, with samples, screenshots of the Info page, app logs, etc.?
